mdx-layout-core/doc/index.mdx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import { mdx } from '@mdx-js/react';
 import { Description } from '~/props';
 import { CoreLayout } from '~/mdx-layout-core';
-export default CoreLayout;
+import { CustomCoreLayout } from '~/doc-layout/src/CustomCoreLayout';
```

Comment:
What about SpeedyCoreLayout ? To keep the name in case of we change the lib underneath ?

Reply:
right now these 3 wrappers are private for "self-testing" dockit-react

export publicly `SpeedyCoreLayout` in addition to `CoreLayout` might be a good idea indeed
do you think such API sells better than asking to read docs in dockit-core about speedy-links?

I might make another PR with this if we agree

Reply:
That was my take too

Reply:
let's say I stop arguing and agree with you
so you say all layout, from dockit-core layout to React and Vue should do it by default and release as a new major?

Reply:
I would say yes.
I don't know if it needs to be a major if there is no API change and it's all internal to `CoreLayout`.
Either way I'm find with this idea.

Reply:
I made it a default